reddit_liveupdate/hooks.py

```diff
@@ -0,0 +1,35 @@
+from pylons import c
+
+from r2.config import feature
+from r2.lib.hooks import HookRegistrar
+from r2.models import DefaultSR, NamedGlobals, NotFound
+
+from reddit_liveupdate.models import LiveUpdateEvent
+from reddit_liveupdate.pages import LiveUpdateHappeningNowBar
+
+hooks = HookRegistrar()
+
+
+@hooks.on("hot.get_content")
+def add_featured(controller):
+    if not feature.is_enabled('live_happening_now'):
+        return None
+
+    # Not on front page
+    if not isinstance(c.site, DefaultSR):
```

Comment:
This `if` by itself would, but this hook (hot.get_content) is only triggered on the `HotController`, so it should be safe. Maybe that should be clearer or I should be more explicit in logic?